clang/unittests/Format/FormatTest.cpp

```diff
@@ -10848,6 +10848,19 @@ TEST_F(FormatTest, WrapsTemplateDeclarations) {
                "public:\n"
                "  E *f();\n"
                "};");
+  AlwaysBreak.RequiresClausePosition = FormatStyle::RCPS_SingleLine;
+  verifyNoChange("template <typename T> requires std::floating_point<T>\n"
+                 "using LerpValue = ClampedValue<T, T{0}, T{1}>;",
+                 AlwaysBreak);
+  AlwaysBreak.RequiresClausePosition = FormatStyle::RCPS_WithPreceding;
+  verifyNoChange("template <typename T> requires std::floating_point<T>\n"
+                 "using LerpValue = ClampedValue<T, T{0}, T{1}>;",
+                 AlwaysBreak);
+  AlwaysBreak.RequiresClausePosition = FormatStyle::RCPS_OwnLine;
+  verifyNoChange("template <typename T>\n"
+                 "requires std::floating_point<T>\n"
```

Comment:
```suggestion
                 "  requires std::floating_point<T>\n"
```
The tests fail, did you run them locally?